resources/packs/github/github.lr

```diff
@@ -381,7 +399,7 @@ private github.commit @defaults("sha") {
 
 // GitHub Repository Pull Request 
 private github.mergeRequest @defaults("id state") {
-  // Pull request ID
+  // Pull request id
```

Comment:
Hey @scottford-io why the lowercase id here and below?

Reply:
I also think ID is upper case

Reply:
agreed!

Reply:
fixed!